### PR TITLE
Homework 1 implemented

### PR DESCRIPTION
**Pull request by amialkumau**
Link to Homework 1 task - https://epa.ms/nodejs19-hw1.

Both tasks were done in Vanila JS. What was done:
* task 1.1
* task 1.2
* added babel
* added documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+**Task Description**: the program should read a string from the standard input `stdin`,
+reverse it and write it to the standard output `stdout`.
+
+**Solution**: to reverse incoming string from `stdin` I'm using `Transform` stream.
+
+**How to launch**: run this command in your terminal
+```bash
+npm run start:task1
+```
+It will parse program's code using `babel` and output it to the `dist/task1` folder.
```
Comment (boccob) [inline README.md]: Documentation is already outdated, there is no `babel` 🙀

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
-# EPAM-NodeJS-Training
\ No newline at end of file
+# EPAM-NodeJS-Training
```
Comment (boccob) [inline README.md:1]: Thanks for readme!

File: Homework-1-NodeJS-Fundamentals/task1/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Transform } from 'stream';
+
+const reverseString = str => str.split("").reverse().join("");
+
+const removeLineBreaks = str => str.replace(/(\r\n|\n|\r)/gm, "");
```
Comment (boccob) [inline Homework-1-NodeJS-Fundamentals/task1/index.js]: I would use `trim()` but thanks for thinking about different types of line separators.

File: Homework-1-NodeJS-Fundamentals/task2/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        callback(null, `${JSON.stringify(chunk)}\n`);
+    }
+});
+
+/**
+ * @desc it's possible to ignore Amount column from parsing not to have it in output file
+ */
+csv({ignoreColumns: /Amount/})
+    .fromFile(csvFilePath)
+    .subscribe((json,lineNumber)=>{
```
Comment (boccob) [inline Homework-1-NodeJS-Fundamentals/task2/index.js]: I don't get why you need `subscribe` that does nothing.

File: Homework-1-NodeJS-Fundamentals/task2/index.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        return new Promise((resolve) => {
+            resolve();
+        })
+    })
+    .pipe(writable1)
+    .on('error', err => console.log(`Something went wrong! Error: ${err.message}`))
+
+/**
+ * @desc transform stream is used to get rid of 'Amount' property in output file
```
Comment (boccob) [inline Homework-1-NodeJS-Fundamentals/task2/index.js:35]: You can also transform property names to lowercase :)

File: Homework-1-NodeJS-Fundamentals/task2/index.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+ */
+csv()
```
Comment (boccob) [inline Homework-1-NodeJS-Fundamentals/task2/index.js:37]: Can you maybe add an example with usage of `pipeline`? It won't be much different but it's a part of requirements.

**Review by boccob — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amialkumau) [reply README.md]: Removed this part of the doc :)

File: Homework-1-NodeJS-Fundamentals/task1/index.js (hunk 3 above)
Comment (amialkumau) [reply Homework-1-NodeJS-Fundamentals/task1/index.js]: Agree, this is more elegant way. Forgot that trim removes all the line terminators also.
Thanks!

File: Homework-1-NodeJS-Fundamentals/task2/index.js (hunk 4 above)
Comment (amialkumau) [reply Homework-1-NodeJS-Fundamentals/task2/index.js]: Yeah, understand that it looks weird, but according to the task we should read/write file content line by line. This approach provides line by line reading, in that `subscribe` we can also perform some action/transform, but by the task no need to do it.
https://github.com/Keyang/node-csvtojson#asynchronously-process-each-line-from-csv-url

File: Homework-1-NodeJS-Fundamentals/task2/index.js (hunk 5 above)
Comment (amialkumau) [reply Homework-1-NodeJS-Fundamentals/task2/index.js:35]: Agree, thanks! Didn't notice that in expected format keys are in lower case. Nice catch!

File: Homework-1-NodeJS-Fundamentals/task2/index.js (hunk 6 above)
Comment (amialkumau) [reply Homework-1-NodeJS-Fundamentals/task2/index.js:37]: Added example with `pipeline`

File: Homework-1-NodeJS-Fundamentals/task2/index.js (hunk 4 above)
Comment (boccob) [reply Homework-1-NodeJS-Fundamentals/task2/index.js]: I think even without `subscribe` it processes files line by line in stream mode. You can verify it via adding some transform stream that will log every chunk in console.

**Review by boccob — APPROVED** [review]

**Pull request merged**